packages/client/src/orders/types/orderShippingAddressChangeRequests.types.ts

```diff
@@ -0,0 +1,31 @@
+import type { CheckoutAddress } from '../../types';
+import type { CheckoutOrder } from '../../checkout';
+import type { Order } from './order.types';
+
+export type OrderShippingAddressChangeRequests =
+  Array<OrderShippingAddressChangeRequest>;
+
+export type OrderShippingAddressChangeRequest = {
+  id: string;
+  checkoutOrderId: CheckoutOrder['id'];
+  orderId: Order['id'];
+  status: OrderShippingAddressChangeRequestStatus;
+  createdDate: string;
+  updatedDate: string;
+  merchantOrders: Array<MerchantOrder>;
+  shippingAddress: CheckoutAddress;
+  isForcedUpdate: boolean;
+};
+
+export enum OrderShippingAddressChangeRequestStatus {
+  Requested,
+  Pending,
+  Approved,
+  Denied,
+  TimedOut,
+  Cancelled,
+}
+
+export type MerchantOrder = {
```

Comment:
There are the types `MerchantOrderNormalized` and `MerchantOrderDenormalized` already and I think the name of this type might raise some confusion among consumers of the package. Since the type only has one property in it, what if we have an anonymous type instead like this:

```js
export type OrderShippingAddressChangeRequest = {
  id: string;
  checkoutOrderId: CheckoutOrder['id'];
  orderId: Order['id'];
  status: OrderShippingAddressChangeRequestStatus;
  createdDate: string;
  updatedDate: string;
  merchantOrders: Array<{ id: string}>;
  shippingAddress: CheckoutAddress;
  isForcedUpdate: boolean;
};

```

Reply:
Based on the previous comment, that type is no longer an array, so I used the anonymous type without the array part.